src/lib/Config.js

```diff
@@ -58,6 +58,20 @@ class Config {
             }
         }
 
+        this.healthChecks = { allowFrom: ['127.0.0.1/8', '::1'] };
+        if (config.healthChecks && config.healthChecks.allowFrom) {
+            assert(config.healthChecks.allowFrom instanceof Array,
```

Comment:
I was reading up on it and it was because browsers run into weird scoping issues due to the global `window` object. Either way, I will make the change as it's a good practice.